docs/content/en/docs/whatsnew/changelog.md

```diff
@@ -41,6 +41,45 @@ description: >
 * When upgrading to a new minor version, a new OS image must be created using the new image-builder CLI pertaining to that release.
 {{% /alert %}}
 
+## [v0.23.1](https://github.com/aws/eks-anywhere/releases/tag/v0.23.1)
+
+### Supported OS version details
+|                     | vSphere | Bare Metal | Nutanix | CloudStack | Snow |
+|:-------------------:|:-------:|:----------:|:-------:|:----------:|:----:|
+|    Ubuntu 20.04     |    ✔    |     ✔      |    ✔    |     —      |  ✔   |
+|    Ubuntu 22.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+| Bottlerocket 1.32.0 |    ✔    |     —      |    —    |     —      |  —   |
+|      RHEL 8.x       |    ✔    |     ✔      |    ✔    |     ✔      |  —   |
+|      RHEL 9.x       |    ✔    |     ✔      |    ✔    |     ✔      |  —   |
+
+\* RHEL 8's kernel version (4.18) is not supported by kubeadm for Kubernetes versions 1.32 and above (see Kubernetes GitHub issue [#129462](https://github.com/kubernetes/kubernetes/issues/129462)). As a result, EKS Anywhere does not support using RHEL 8 as the node operating system for Kubernetes versions 1.32 and above.
+
+### Added
+- Add certificate component renewal for Ubuntu, Rhel and Bottlerocket ([#9939](https://github.com/aws/eks-anywhere/pull/9939))
```

Comment:
```suggestion
- Add certificate renewal command for control-plane and external etcd components for Ubuntu, Rhel and Bottlerocket OS [#9781](https://github.com/aws/eks-anywhere/pull/9781),[#9782](https://github.com/aws/eks-anywhere/pull/9782)
```
Since there are many PRs, we can include 2 main PRs and can skip others.